addon/components/ember-tooltip-base.js

```diff
@@ -554,3 +559,27 @@ export default Component.extend({
     }
   }
 });
+
+function mergeModifiers(defaults, overrides = {}) {
```

Comment:
Could we use [Ember's `merge()` method](https://emberjs.com/api/api/ember/release/functions/@ember%2Fpolyfills/merge) instead?

Reply:
Or failing that, using the spread operator [detailed here](https://davidwalsh.name/merge-objects)?

Reply:
`Ember.merge` is deprecated in ember-beta (3.5, I believe). The problem with both is that they don't do deep merges, which is what we'd need to preserve defaults that are not overridden. This is a pretty naïve deep merge implementation (only depth=1), but we can totally refactor it and/or swap in a good deep merge library later, if our needs change.